### DISCUSSION TITLE
option --disk-cache-dir is listed but not honored/enabled

### DISCUSSION THREAD

**Discussion by bchill** [oobabooga/text-generation-webui#4347] [Q&A]
Even if I start the server with the `--disk-cache-dir /tmp/cache`, it insists on opening up the cache dir inside the text-generation-webui dir, which, in this case, isn't owned by the user starting the server:

> Traceback (most recent call last):
>   File "/opt/text-generation-webui/default/server.py", line 230, in <module>
>     create_interface()
>   File "/opt/text-generation-webui/default/server.py", line 141, in create_interface
>     extensions_module.create_extensions_block()  # Extensions block
>     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
>   File "/opt/text-generation-webui/0.00/modules/extensions.py", line 192, in create_extensions_block
>     extension.ui()
>   File "/opt/text-generation-webui/0.00/extensions/gallery/script.py", line 96, in ui
>     samples=generate_html(),
>             ^^^^^^^^^^^^^^^
>   File "/opt/text-generation-webui/0.00/extensions/gallery/script.py", line 71, in generate_html
>     image_html = f'<img src="file/{get_image_cache(path)}">'
>                                    ^^^^^^^^^^^^^^^^^^^^^
>   File "/opt/text-generation-webui/0.00/modules/html_generator.py", line 187, in get_image_cache
>     img.convert('RGB').save(output_file, format='PNG')
>   File "/opt/python/default/lib/python3.11/site-packages/PIL/Image.py", line 2350, in save
>     fp = builtins.open(filename, "w+b")
>          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
> PermissionError: [Errno 13] Permission denied: 'cache/cache_Example.png.png'
> 

Am I missing something?

**Comment by kdubey22** [comment]
Were you able to resolve it?

**Reply by bchill** [reply]
It was fixed in an update:

https://github.com/oobabooga/text-generation-webui/issues/4359
